lib/service.js

```diff
@@ -403,6 +403,31 @@ AWS.Service = inherit({
   /**
    * @api private
    */
+  getSkewCorrectedDate: function getSkewCorrectedDate() {
+    return new Date(new Date().getTime() + this.config.systemClockOffset);
```

Comment:
Since `new Date().getTime()` is the equivalent to `Date.now()`, can you switch your usage? `Date.now()` actually ends up being [way faster](https://jsperf.com/date-now-vs-date-gettime) depending on your browser